apps/condo/domains/ticket/schema/PredictTicketClassificationService.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Generated by `createservice ticket.PredictTicketClassificationService --type queries`
+ */
+const { getById, GQLCustomSchema } = require('@core/keystone/schema')
+const { canPredictTicketClassification } = require('@condo/domains/ticket/access/PredictTicketClassificationService')
+const { GQLError, GQLErrorCode: { INTERNAL_ERROR } } = require('@core/keystone/errors')
+const { NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const ML_SPACE = process.env.ML_SPACE_TICKET_CLASSIFIER ? JSON.parse(process.env.ML_SPACE_TICKET_CLASSIFIER) : {}
+const fetch = require('node-fetch')
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const errors = {
+    TICKET_RULE_NOT_FOUND: {
+        query: 'predictTicketClassification',
+        code: INTERNAL_ERROR,
+        type: NOT_FOUND,
+        message: 'ticket classifier rule not found',
+        messageForUser: 'api.user.predictTicketClassification.TICKET_RULE_NOT_FOUND',
+    },
+    ML_SPACE_NOT_CONFIGURED: {
+        query: 'predictTicketClassification',
+        code: INTERNAL_ERROR,
+        type: NOT_FOUND,
+        message: 'ML_SPACE_TICKET_CLASSIFIER env variable needs to have endpoint, authKey, workspace',
+        messageForUser: 'api.user.predictTicketClassification.ML_SPACE_NOT_CONFIGURED',
+    },
+}
+
+const PredictTicketClassificationService = new GQLCustomSchema('PredictTicketClassificationService', {
+    types: [
+        {
+            access: true,
+            type: 'input PredictTicketClassificationInput { details: String! }',
+        },
+    ],
+    queries: [
+        {
+            access: canPredictTicketClassification,
+            schema: 'executePredictTicketClassification (data: PredictTicketClassificationInput!): TicketClassifierRule',
```

Comment:
Just ‘predict ticket classification’
```suggestion
            schema: 'predictTicketClassification (data: PredictTicketClassificationInput!): TicketClassifierRule',
```